### ISSUE TITLE
Unit Test for CurrencyPair - iso string parse FAILS

### ISSUE THREAD

**Issue by devsi** [moneyphp/money#695]
Running the unit tests against the built lib, all pass but 1. 

```
There was 1 failure:

1) Tests\Money\CurrencyPairTest::it_parses_an_iso_string
assert(is_numeric($matches[2])) in /app/src/CurrencyPair.php:65

FAILURES!
Tests: 1790, Assertions: 2599, Failures: 1.
```

Should be a fairly straight forward fix to get this to pass. I'll take a look myself if I get time and submit a PR. Otherwise leaving this here for a contributor to pick up!

Notable Files@Methods:

src/CurrencyPair@createFromIso
tests/CurrencyPairTest@it_parses_an_iso_string

Thanks :)

**Comment by frederikbosch** [comment]
Why does it fail for you, and is the test suite not failing?

**Comment by frederikbosch** [comment]
Closed by #700
